libs/sdk-backend-bear/src/backend/workspace/attributes/index.ts

```diff
@@ -91,17 +93,38 @@ export class BearWorkspaceAttributes implements IWorkspaceAttributesService {
         refs: ObjRef[],
     ): Promise<IAttributeDisplayFormMetadataObject[]> => {
         const displayFormUris = await objRefsToUris(refs, this.workspace, this.authCall);
-        const wrappedAttributeDisplayForms = await this.authCall((sdk) =>
-            sdk.md.getObjects<GdcMetadata.IWrappedAttributeDisplayForm>(this.workspace, displayFormUris),
-        );
-        return wrappedAttributeDisplayForms.map(
-            (
-                wrappedDisplayForm: GdcMetadata.IWrappedAttributeDisplayForm,
-            ): IAttributeDisplayFormMetadataObject => {
-                const displayFormDetails = wrappedDisplayForm.attributeDisplayForm;
-                return this.buildAttributeDisplayForm(displayFormDetails);
-            },
-        );
+        if (uniq(displayFormUris).length !== refs.length) {
+            throw new UnexpectedError(
+                "DUPLICATED_REFERENCED_OBJECT",
```

Comment:
SPI should not depend on this validation (should be handled in the app)

Reply:
I have removed it.